package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-helmet",
   "description": "A document head manager for React",
-  "version": "5.2.1",
+  "version": "5.2.2",
```

Comment:
It can't be a patch upgrade, a new version of `react-side-effect` uses react >= 16.9

https://github.com/nfl/react-helmet/pull/501#issuecomment-577104900

Reply:
Okay understood, so need to create different pull react for future minor version or you are already added into development plan? Asking because I am not seeing any label on this.

Reply:
I've just noticed that upgrade of react-side-effect breaks backward compatibility (I’m not a maintainer of this repo)

Personally I would like to help to resolve react depreciations, but it looks like `react-helmet-async` appeared for a reason with the same intentions

Reply:
You're right, and @wheeler [mentioned so already](https://github.com/nfl/react-helmet/pull/472#issuecomment-535236214) few months ago, however since this package is no longer maintained, I switched to `react-helmet-async` and looks like everyone is doing the same.

<img width="1111" alt="Screen Shot 2020-01-23 at 08 59 00" src="https://user-images.githubusercontent.com/4334778/72966221-a0424680-3dbe-11ea-9e6c-880c19c5b148.png">

Reply:
I already added in my development plan, soon I will be switched to react-helmet-async. Thanks @maxmalov & @rifaidev for quick response.

Reply:
I switched today - it's fairly simple for a client-side rendered React project, you just have to add the provider and then replace all imports and that's pretty much it.